devicetypes/qubino/qubino-dimmer.src/qubino-dimmer.groovy

```diff
@@ -363,6 +363,8 @@ private dimmerEvents(physicalgraph.zwave.Command cmd, ep = null) {
 		result << createEvent(name: "level", value: cmdValue == 99 ? 100 : cmdValue)
 	}
 
+	sendHubCommand(encapCommands(getPowerMeterCommands()))
```

Comment:
@MWierzbinskaS Is it needed for all Qubino dimmers? As far as I know Qubino Flush Dimmer and Qubino Flush Dimmer 0-10V are reporting power meter values [W] itself.

Reply:
@KKlimczukS I moved this to BasicReport handling as we spoke